css/src/components/pagination.scss

```diff
@@ -0,0 +1,194 @@
+$font-size-lg: $font-size-6 !default;
+$font-size-sm: $font-size-8 !default;
+
+$pagination-color: $text !default;
+$pagination-border-color: $border !default;
+$pagination-margin: 0.25rem !default;
+
+$pagination-hover-color: $primary-hover !default;
+$pagination-hover-border-color: $primary-hover !default;
+
+$pagination-focus-color: $primary-active !default;
+$pagination-focus-border-color: $primary-active !default;
+
+$pagination-active-color: $primary-active !default;
+$pagination-active-border-color: $primary-active !default;
+
+$pagination-disabled-color: $secondary !default;
+$pagination-disabled-background-color: $secondary-background !default;
+$pagination-disabled-border-color: $secondary-background !default;
+
+$pagination-current-color: $text-invert !default;
+$pagination-current-background-color: $primary !default;
+$pagination-current-background-color-hover: $primary-hover !default;
+$pagination-current-border-color: $text-invert !default;
+
+$pagination-ellipsis-color: $secondary-background !default;
+
+$pagination-shadow-inset: inset 0 1px 2px $box-shadow-color-light;
+
+.pagination {
+	flex-wrap: wrap;
+	margin: -$pagination-margin;
+
+	.pagination-center {
+		@include orientation-landscape {
+			.pagination-previous {
+				order: 1;
+			}
+
+			.pagination-list {
+				justify-content: center;
+				order: 2;
+			}
+
+			.pagination-next {
+				order: 3;
+			}
+		}
+	}
+
+	// Sizes
+
+	&.pagination-sm {
+		font-size: $font-size-sm;
+	}
+
+	&.pagination-lg {
+		font-size: $font-size-lg;
+	}
+}
+
+.pagination-list {
```

Comment:
Are all of the `pagination-` classes subclasses of `.pagination` ? Should they be nested under `.pagination`?